systest/integration2/multitenancy_test.go

```diff
@@ -0,0 +1,136 @@
+//go:build integration2
+
+/*
+ * Copyright 2024 Dgraph Labs, Inc. and Contributors *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/dgraph-io/dgo/v240/protos/api"
+	"github.com/dgraph-io/dgraph/v24/dgraphapi"
+	"github.com/dgraph-io/dgraph/v24/dgraphtest"
+	"github.com/dgraph-io/dgraph/v24/x"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	gc *dgraphapi.GrpcClient
+	hc *dgraphapi.HTTPClient
+)
+
+func addData(t *testing.T, gc *dgraphapi.GrpcClient) {
+	require.NoError(t, gc.SetupSchema(`name: string  .`))
+
+	rdfs := `
+	_:a <name> "alice" .
+	_:b <name> "bob" .
+	_:c <name> "sagar" .
+	_:d <name> "ajay" .`
+	_, err := gc.Mutate(&api.Mutation{SetNquads: []byte(rdfs), CommitNow: true})
+	require.NoError(t, err)
+}
+
+func TestRunAllMultiTenancy(t *testing.T) {
+	conf := dgraphtest.NewClusterConfig().WithNumAlphas(1).WithNumZeros(1).WithReplicas(1).WithACL(time.Hour)
+	c, err := dgraphtest.NewLocalCluster(conf)
+	require.NoError(t, err)
+	defer func() { c.Cleanup(t.Failed()) }()
+	require.NoError(t, c.Start())
+	var cleanup func()
+	gc, cleanup, err = c.Client()
+	require.NoError(t, err)
+	defer cleanup()
+	require.NoError(t, gc.LoginIntoNamespace(context.Background(),
+		dgraphapi.DefaultUser, dgraphapi.DefaultPassword, x.GalaxyNamespace))
+
+	hc, err = c.HTTPClient()
+	require.NoError(t, err)
+	require.NoError(t, hc.LoginIntoNamespace(dgraphapi.DefaultUser,
+		dgraphapi.DefaultPassword, x.GalaxyNamespace))
+	addData(t, gc)
+
+	RunAll(t)
+}
+
+func RunAll(t *testing.T) {
+	t.Run("login into the ns after drop data opration from the ns", testDropDataNSLogin)
+	t.Run("login into the default ns after drop data opration from default ns", testDropDataNSLogin)
```

Comment:
what about logging into ns after drop data operation from default ns?

Reply:
in the second test, I am logging into all namespaces after drop_data OP. I should rename the test.